test/test.js

```diff
@@ -376,6 +376,21 @@ suite('Response', function() {
       return json;
     })
   })
+
+  test('error creates error Response', function() {
+    var r = Response.error()
+    assert(r instanceof Response)
+    assert.equal(r.status, 0)
+    assert.equal(r.statusText, '')
+    assert.equal(r.type, 'error')
+  })
+
+  test('redirect creates redirect Response', function() {
+    var r = Response.redirect('/hello', 301)
+    assert(r instanceof Response);
+    assert.equal(r.status, 301)
+    assert.equal(r.headers.get('Location', '/hello'), '/hello')
```

Comment:
Yes, I forgot to mention that. They pass in Chrome, Firefox, and Safari. I did not manually test IE/Edge or any mobile browsers. I also manually inspected the `Response` objects created by Chrome and Firefox to verify that the instances created here look similar in terms of data not mentioned by the spec.